babel.config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "14"
```

Comment:
is it ok to have min node version set separately for each module?

Reply:
I'm not sure what you're asking _in this context_, but generally speaking our minimum version is 14.  in this case, it's `@appium/support`'s minimum version, and that's a dependency of `teen_process`, so it can't be any older.  Each project (not package) will necessarily have its own Babel config.